liberty-maven-plugin/src/it/ear-project-it/SampleEAR/pom.xml

```diff
@@ -89,7 +90,7 @@
                     </assemblyArtifact>
                     <serverName>test</serverName>
                     <appsDirectory>apps</appsDirectory>
-                    <looseApplication>true</looseApplication>
+                    <looseApplication>false</looseApplication>
```

Comment:
Are there any other test cases for ear applications with `looseApplication` set to `true`? If not, can we add a new test case for this scenario?

Reply:
Forgot I had done this. Good catch 👍